### ISSUE TITLE
Search Rooms and Playlists input is lacking of colour contrast

### ISSUE THREAD

**Issue by Ckrapock** [mchev/blinest#67]
A follow up from this one : https://github.com/mchev/blinest/issues/64

The input from users in search rooms and playlist is lacking some contrast.

![image](https://user-images.githubusercontent.com/125498940/227334467-63248892-7b80-483e-82a6-ab367e9f0da2.png)

![image](https://user-images.githubusercontent.com/125498940/227334560-e3e1ac17-9c8c-48ad-9aae-caf48cbf20be.png)

**Comment by Ckrapock** [comment]
Thanks!!!
